lib/pbench/test/unit/agent/task/test_copy_result_tb.py

```diff
@@ -22,9 +22,10 @@ def config_and_logger(self, valid_config):
 
     @responses.activate
     def test_copy_tar(self, valid_config):
+        tbname = os.path.basename(tarball)
         responses.add(
             responses.PUT,
-            "http://pbench.example.com/api/v1/upload/ctrl/controller",
+            f"http://pbench.example.com/api/v1/upload/ctrl/controller/{tbname}",
```

Comment:
Looks increasingly odd every time I see the pattern. Either we should add a "guide word" like the "ctrl" for the name, or we should remove "ctrl" and streamline things. The inconsistency bugs me.

E.g., `/api/v1/upload/ctrl/<controller>/name/<filename>` or `/api/v1/upload/<controller>/<filename>`.

Reply:
I don't like the extra "guide word".  And, see my previous comment about `ctrl`.

Reply:
Moved to #2287.